src/dftbp/api/mm/dftbplus.h

```diff
@@ -258,6 +258,16 @@ void dftbp_set_coords_lattice_origin(DftbPlus *instance, const double *coords,
 int dftbp_get_nr_atoms(DftbPlus *instance);
 
 
+/**
+ * Queries the nr. of k-points in the system.
+ *
+ * \param[inout] instance Handler of the DFTB+ instance.
+ *
+ * \return Nr. of k-points
+ */
+int dftbp_get_nr_kpoints(DftbPlus *instance);
```

Comment:
This way, the name C-name is inconsistent with the Fortran one....
```suggestion
int dftbp_nr_kpoints(DftbPlus *instance);
```
We really should somehow make the names in the Fortran an C interface consistent, probably I'll make an additional separate PR on this.

Reply:
Changed